resize-observer/devicepixel2.html

```diff
@@ -0,0 +1,152 @@
+<!doctype html>
+<link rel="match" href="devicepixel2-ref.html">
+<meta name="assert" content="Device pixel content box sizes and pixel snapping are correct in Resize Observer callback">
```

Comment:
As atotic noted, "correctness" is sort of undefined for pixel-snapping, so it feels a little like over-promising to say that we're checking for "correct" pixel-snapping.

Maybe better/more-precise to say something like this:
> "Resize Observer's reported device pixel content box size should be consistent with the actual pixel-snapped painted box size"